Scripts/Proxies/Vive/ViveProxy.cs

```diff
@@ -58,12 +59,21 @@ public override void Awake()
 #endif
         }
 
-        static void PostAnimate(ProxyHelper.ButtonObject[] buttons, Dictionary<Transform, ProxyAnimator.TransformInfo> transformInfos, ActionMapInput input)
+        static void PostAnimate(Affordance[] affordances, AffordanceDefinition[] affordanceDefinitions, Dictionary<Transform, ProxyAnimator.TransformInfo> transformInfos, ActionMapInput input)
         {
             var proxyInput = (ProxyAnimatorInput)input;
-
-            foreach (var button in buttons)
+            foreach (var button in affordances)
             {
+                AffordanceAnimationDefinition affordanceAnimationDefinition = null;
+                foreach (var definition in affordanceDefinitions)
+                {
+                    if (definition.control == button.control)
+                    {
+                        affordanceAnimationDefinition = definition.animationDefinition;
+                        break;
+                    }
+                }
```

Comment:
I'd like to refactor this into a dictionary, or a list of Tuples to avoid having to do this lookup here. Does that make sense, or will you have to do this n^2 task beforehand to match up the control IDs?

This makes me think that we should do this mapping at Start() and always deal with a Dictionary of control-to-definition or list of definitions.

Reply:
Just pushed list of Tuple <ProxyFeedbackRequest, Coroutine> refactor to THPB.  Added an alias too.